src/pages/DistributePage/AllocationsTable.tsx

```diff
@@ -30,10 +44,10 @@ const AllocationTable = ({ users }: { users: IUser[] }) => {
         {
           label: 'Name',
           accessor: 'name',
-          render: function UserName(u: IUser) {
+          render: function UserName(u: IAllocateUser) {
             return (
               <div className={classes.avatarCell}>
-                <ApeAvatar user={u} className={classes.avatar} />
+                <NewApeAvatar name={u.name} className={classes.avatar} />
```

Comment:
Good opportunity here to switch this over to using Stitches. Pretty close to having this whole file have no dependency on material-ui

Reply:
^ attn @f1lander